utils/documentOps.py

```diff
@@ -142,6 +174,26 @@ def print_row(array):
         str += a + " |"
     print(str)
 
+# Min opset: if explicit ==MIN== value, use as is. Otherwise min of default and
+# custom versions delivered in opset_dict.
+def compute_min_opset(op):
+    if op in min_dict:
```

Comment:
What about the operations that only support one version? For example `MatMulInteger` only has version 10 available in `ONNX`. So we should put MIN=NA and MAX=10. We should account for this situation where only one version is givenl and put NA for the MIN versions.

Reply:
You are not understanding my example..... MatMulInteger ONLY has version 10. There is no 9 or 11. As of today there is only one version available. If you search MatMulInteger for ONNX via the documentation you will see what I mean. I think the point I am saying is if for specific operation there is only support for one version (no previous version and no newer version). Should we make the MIN=10 and MAX=NA and if there is a new version that eventually comes out then we just update the MAX to the newer version.